h5glance/terminal.py

```diff
@@ -109,30 +110,39 @@ def object_node(self, obj, name):
         detail = attr_detail = ''
         color_start = color_stop = ''
 
-        if self.expand_attrs:
-            children += attrs_tree_nodes(obj)
-        else:
-            n = len(obj.attrs)
-            if n:
-                attr_detail = ' ({} attributes)'.format(n)
+        obj_id = h5py.h5o.get_info(obj.id).addr
+        first_link = self.visited.setdefault(obj_id, obj.name)
+        is_first_link = first_link == obj.name
+
+        if is_first_link:
```

Comment:
I think it would be neater here to have an `if not is_first_link` with an early return, and then leave the code further down as it was.

Reply:
The reasons are `color_start`, `color_stop`.

Reply:
I see them. I think it's neater to move those assignments up before the `is_first_link` check. If it's not clear what I mean, I'm happy to do the rearrangemnent myself.

Reply:
Please go ahead, -- and thank you for your collaboration!